docs/scos/dev/migration-program/migration-to-paas/paas-assessment-documents/is-project-based-on-or-updated-to-the-202108-0-demoshop-release.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Is project based on or updated to the 202108.0 demoshop release?
+description: This document allows you to assess if a project is based on or updated to the 202108.0 demoshop release.
+template: howto-guide-template
+---
+
+
+`202108.0` is the earliest release compatible with the latest `docker/sdk`, so it’s the earliest release to update to.
```

Comment:
In the intro, we are saying that the reader is going to check whether the project is of version 202108.0. But to me the doc explains only how to estimate the upgrade of the modules related to `Separate endpoint bootstraps`. Does it make sense to rename the doc into "Check if separate endpoint bootstraps are integrated"?

Reply:
I think you are right. It is a more historical title because initially, we had a requirement for projects to be upgraded at least to 202108.0, but then we figured out that it is not necessary and to save time now we require only a few modules to be upgraded, to be able to integrate the Separate endpoint bootstraps feature.